app/components/unregistered-contributor-form.js

```diff
@@ -0,0 +1,45 @@
+import Ember from 'ember';
+
+import {
+  validator, buildValidations
+}
+from 'ember-cp-validations';
+
+/**
+* Validations for adding unregistered contributor form.  fullName must be present
+* and have three letters, and the username (email) must be present and of appropriate format.
+*/
+const Validations = buildValidations({
+    fullName: {
+        description: 'Full Name',
+        validators: [
+            validator('presence', true),
+            validator('length', {
+                min: 3
+            })
+        ]
+    },
+    username: {
+        validators: [
+            validator('presence', true),
+            validator('format', {
+                type: 'email'
+            })
+       ]
+    },
+});
+
+export default Ember.Component.extend(Validations, {
+    fullName: null,
+    username: null,
+    isFormValid: Ember.computed.alias('validations.isValid'),
+    actions: {
+        emptyView() {
```

Comment:
You're right I don't think I need it here - I'm just learning that you don't have to pass in the actions at every level.  I used to do sendAction at every level of every nested component.  It's pretty cool that you can just pass the action name.